doc/make.py

```diff
@@ -206,15 +207,15 @@ def _add_redirects(self):
 
                 try:
                     title = self._get_page_title(row[1])
-                except Exception:
+                except docutils.io.InputError:
```

Comment:
I don't think it is worth trying to pin down the exception, as explained in the comment below there are several cases where this may trigger an error. Why is that `except Exception` that bad?

Reply:
> Why is that except Exception that bad?

In this docs file it is pretty benign, and the motivation is largely "because it shows up whenever I grep for 'except Exception'".